tests/loops/ test_loop_state_dict.py

```diff
@@ -0,0 +1,47 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
There is leading whitespace in the filename here, haha, how is this possible 😄

Reply:
ahaha removed !